packages/flutter/test/material/theme_test.dart

```diff
@@ -376,15 +376,15 @@ void main() {
 
     List<TextStyle> extractStyles(TextTheme textTheme) {
       return <TextStyle>[
-        textTheme.display4,
-        textTheme.display3,
-        textTheme.display2,
-        textTheme.display1,
-        textTheme.headline,
-        textTheme.title,
-        textTheme.subhead,
-        textTheme.body2,
-        textTheme.body1,
+        textTheme.headline1,
+        textTheme.headline2,
+        textTheme.headline3,
+        textTheme.headline4,
+        textTheme.headline5,
+        textTheme.headline6,
+        textTheme.subtitle1,
+        textTheme.bodyText1,
+        textTheme.bodyText2,
```

Comment:
Is subtitle(14)/subtitle2(18) missing here?

Reply:
This test just checks the original 2014 text styles (still the default). It's now a regression test. Will add a comment to that effect.